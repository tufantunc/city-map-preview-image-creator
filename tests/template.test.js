const template = require('../template.js');

describe('template function', () => {
    it('should return page template', () => {
        const pageTemplate = template();
        expect(pageTemplate).toMatchSnapshot();
    });
});