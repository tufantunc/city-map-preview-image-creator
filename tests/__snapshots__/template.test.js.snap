// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`template function should return page template 1`] = `
"<!DOCTYPE html>
        <html>
            <head>
                <meta charset=\\"UTF-8\\">
                <style>
                    * {
                        margin: 0;
                        padding: 0;
                    }
                    html, body {
                        width: 100%;
                        height: 100%;
                    }
                    #map {
                        display: block;
                        width: 100%;
                        height: 100%;
                    }
                </style>
                <script src=\\"https://api-maps.yandex.ru/2.1/?lang=tr_TR\\" type=\\"text/javascript\\"></script>
                <script type=\\"text/javascript\\">
                    ymaps.ready(init);
                    var myMap;

                    function init(){     
                        myMap = new ymaps.Map(\\"map\\", {
                            center: [41.008857, 28.96747],
                            zoom: 9,
                            controls: []
                        });
                    }
                </script>
            </head>
            <body>
                    <div id=\\"map\\"></div>
            </body>
        </html>"
`;
